src/components/core/DpBulkEditHeader.stories.tsx

```diff
@@ -0,0 +1,96 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpBulkEditHeader from './DpBulkEditHeader.vue'
+
+const meta: Meta<typeof DpBulkEditHeader> = {
+    component: DpBulkEditHeader,
+    title: 'Components/BulkEditHeader',
+    render: (args) => ({
+        components: {
+            DpBulkEditHeader
+        },
+        computed: {
+            selectedItemsCount () {
+                return args.options.filter(checked => checked.checked == true)
+            }
+        },
+        setup() {
+            const toggleSelectAll = () => {
+                for (let element of args.options) {
+                    if(element.checked === false) {
+                        element.checked = true
+                    } else {
+                        element.checked = false
+                    }
```

Comment:
This logic inverts the checked state for each element, instead of just setting checked to true for each element.

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/615/commits/e241f51b2ce6408cc249b8fca407a82e0549c68b